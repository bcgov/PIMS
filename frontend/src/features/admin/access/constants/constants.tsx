import { ColumnWithProps } from 'components/Table';
import { AccessStatusDisplayMapper } from 'constants/accessStatus';
import * as React from 'react';
import { Link } from 'react-router-dom';
import { CellProps } from 'react-table';

import { RowActions } from '../components/RowActions';
import { IAccessRequestModel } from '../interfaces';

export const columnDefinitions: ColumnWithProps<IAccessRequestModel>[] = [
  {
    Header: 'IDIR/BCeID',
    accessor: 'username',
    align: 'left',
    Cell: (props: CellProps<IAccessRequestModel>) => {
      return (
        <Link to={`/admin/user/${props.row.original.userId}`}>{props.row.original.username}</Link>
      );
    },
  },
  {
    Header: 'First name',
    accessor: 'firstName',
    align: 'left',
    clickable: true,
  },
  {
    Header: 'Last name',
    accessor: 'lastName',
    align: 'left',
    clickable: true,
  },
  {
    Header: 'Email',
    accessor: 'email',
    align: 'left',
    clickable: true,
    minWidth: 200,
  },
  {
    Header: 'Position',
    accessor: 'position',
    align: 'left',
    clickable: true,
  },
  {
    Header: 'Status',
    accessor: 'status',
    align: 'left',
    clickable: true,
    width: 100,
    Cell: (props: CellProps<IAccessRequestModel>) =>
      AccessStatusDisplayMapper[props.row.original.status],
  },
  {
    Header: 'Agency',
    accessor: 'agency',
    align: 'left',
    clickable: true,
    minWidth: 200,
  },
  {
    Header: 'Role',
    accessor: 'role',
    align: 'left',
    clickable: true,
    minWidth: 200,
  },
  {
    Header: ' ',
    Cell: RowActions,
    width: 75,
  },
];
