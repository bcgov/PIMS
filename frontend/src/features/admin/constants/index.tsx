import * as React from 'react';
import { ColumnWithProps } from 'components/Table';
import { CellProps } from 'react-table';
import { Link } from 'react-router-dom';
import { IAgency, IAgencyRecord } from 'interfaces';

export const columnDefinitions: ColumnWithProps<IAgencyRecord>[] = [
  {
    Header: 'Agency name',
    accessor: 'name',
    align: 'left',
    sortable: true,
    clickable: true,
  },
  {
    Header: 'Short name',
    accessor: 'code',
    align: 'left',
    clickable: true,
    sortable: true,
  },
  {
    Header: 'Description',
    accessor: 'description',
    align: 'left',
    clickable: true,
    sortable: true,
  },
  {
    Header: 'Parent Agency',
    accessor: 'parent',
    align: 'left',
    clickable: true,
    Cell: (props: CellProps<IAgency>) => {
      return (
        <Link to={`/admin/agency/${props.row.original.parentId}`}>{props.row.original.parent}</Link>
      );
    },
    sortable: true,
  },
];
