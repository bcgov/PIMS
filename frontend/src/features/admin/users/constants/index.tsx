import { ColumnWithProps } from 'components/Table';
import * as React from 'react';
import { FaCheck, FaTimes } from 'react-icons/fa';
import { Link } from 'react-router-dom';
import { CellProps } from 'react-table';

import { RowActions } from '../components/RowActions';
import { IUserRecord } from '../interfaces/IUserRecord';

export const columnDefinitions: ColumnWithProps<IUserRecord>[] = [
  {
    Header: 'IDIR/BCeID',
    accessor: 'username',
    align: 'left',
    Cell: (props: CellProps<IUserRecord>) => {
      return <Link to={`/admin/user/${props.row.original.id}`}>{props.row.original.username}</Link>;
    },
    sortable: true,
  },
  {
    Header: 'First name',
    accessor: 'firstName',
    align: 'left',
    clickable: true,
    sortable: true,
  },
  {
    Header: 'Last name',
    accessor: 'lastName',
    align: 'left',
    clickable: true,
    sortable: true,
  },
  {
    Header: 'Email',
    accessor: 'email',
    align: 'left',
    clickable: true,
    sortable: true,
    minWidth: 200,
  },
  {
    Header: 'Position',
    accessor: 'position',
    align: 'left',
    clickable: true,
    sortable: true,
  },
  {
    Header: 'Active',
    accessor: 'isDisabled',
    align: 'left',
    clickable: true,
    sortable: true,
    width: 100,
    Cell: (props: CellProps<IUserRecord>) =>
      props.row.original.isDisabled ? <FaTimes /> : <FaCheck />,
  },
  {
    Header: 'Agency',
    accessor: 'agency',
    align: 'left',
    sortable: true,
    clickable: true,
    minWidth: 200,
  },
  {
    Header: 'Roles',
    accessor: 'roles',
    align: 'left',
    clickable: true,
    minWidth: 200,
  },
  {
    Header: 'Last Login',
    accessor: 'lastLogin',
    align: 'left',
    clickable: true,
    minWidth: 100,
  },
  {
    Header: 'Created On',
    accessor: 'createdOn',
    align: 'left',
    clickable: true,
    sortable: true,
    minWidth: 100,
  },
  {
    Header: ' ',
    Cell: RowActions,
    width: 75,
  },
];
