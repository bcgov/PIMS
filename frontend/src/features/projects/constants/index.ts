export * from './AgencyResponses';
export * from './DisposalWorkflows';
export * from './DisposeWorkflowStatus';
export * from './ProjectWorkflowComponents';
export * from './ReviewWorkflowStatus';
export * from './SPPApprovalTabs';
