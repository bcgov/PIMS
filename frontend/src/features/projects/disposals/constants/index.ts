export * from './defaultFilter';
export * from './defaultProjectForm';
