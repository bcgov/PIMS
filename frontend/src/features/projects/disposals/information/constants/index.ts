export * from './AddPropertyColumns';
export * from './PropertyColumns';
