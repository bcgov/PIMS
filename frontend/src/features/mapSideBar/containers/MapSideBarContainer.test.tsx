import React from 'react';
import { Router, Route } from 'react-router-dom';
import { createMemoryHistory } from 'history';
import { render, cleanup, wait } from '@testing-library/react';
import configureMockStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import { Provider } from 'react-redux';

import { act } from 'react-dom/test-utils';
import { ToastContainer } from 'react-toastify';
import MapSideBarContainer from './MapSideBarContainer';
import { noop } from 'lodash';
import * as reducerTypes from 'constants/reducerTypes';
import * as actionTypes from 'constants/actionTypes';
import { IParcel } from 'actions/parcelsActions';
import { mockDetails } from 'mocks/filterDataMock';
import VisibilitySensor from 'react-visibility-sensor';
import { useKeycloak } from '@react-keycloak/web';

jest.mock(
  'react-visibility-sensor',
  (): typeof VisibilitySensor => ({ children, partialVisibility, ...rest }: any) => (
    <div {...rest}>{typeof children === 'function' ? children({ isVisible: true }) : children}</div>
  ),
);

jest.mock('@react-keycloak/web');
(useKeycloak as jest.Mock).mockReturnValue({
  keycloak: {
    userInfo: {
      agencies: ['1'],
      roles: ['admin-properties'],
    },
    subject: 'test',
  },
});

const mockStore = configureMockStore([thunk]);
const getStore = (parcelDetail?: IParcel) =>
  mockStore({
    [reducerTypes.NETWORK]: {
      [actionTypes.GET_PARCEL_DETAIL]: {
        status: 200,
      },
    },
    [reducerTypes.PARCEL]: {
      parcelDetail: {
        parcelDetail: parcelDetail,
        propertyTypeId: 0,
      },
      parcels: [],
      draftParcels: [],
    },
    [reducerTypes.LOOKUP_CODE]: { lookupCodes: [] },
    [reducerTypes.PARCEL]: { parcels: [], draftParcels: [] },
  });

const history = createMemoryHistory();

const renderContainer = ({ store }: any) =>
  render(
    <Provider store={store ?? getStore()}>
      <Router history={history}>
        <ToastContainer
          autoClose={5000}
          hideProgressBar
          newestOnTop={false}
          closeOnClick={false}
          rtl={false}
          pauseOnFocusLoss={false}
        />
        <Route path="/mapView/:id?">
          <MapSideBarContainer refreshParcels={noop} properties={[]} />
        </Route>
      </Router>
    </Provider>,
  );

describe('Parcel Detail MapSideBarContainer', () => {
  // clear mocks before each test
  beforeEach(() => {});
  afterEach(() => {
    history.push({ search: '' });
    cleanup();
  });

  describe('basic data loading and display', () => {
    it('sidebar is hidden', async () => {
      await act(async () => {
        const { container } = renderContainer({});
        expect(container.firstChild).toMatchSnapshot();
      });
    });

    it('Empty parcel sidebar matches snapshot', async () => {
      await act(async () => {
        history.push('/mapview?sidebar=true');
        const { container } = renderContainer({});
        expect(container.firstChild).toMatchSnapshot();
      });
    });

    xit('parcel sidebar snapshot loads by id', async () => {
      await act(async () => {
        history.push('/mapview/1?sidebar=true');
        const { container, findByDisplayValue } = renderContainer({
          store: getStore(mockDetails[0]),
        });
        await findByDisplayValue('000-000-000');
        expect(container.firstChild).toMatchSnapshot();
      });
    });

    it('removes the parcel id when the sidebar is closed', () => {
      history.push('/mapview/1?sidebar=false');
      renderContainer({
        store: getStore(mockDetails[0]),
      });
      wait(() => expect(history.location.pathname).toEqual('/mapview'));
    });

    it('removes the parcel id when the sidebar is closed', () => {
      history.push('/mapview/1?sidebar=false');
      renderContainer({
        store: getStore(mockDetails[0]),
      });
      wait(() => expect(history.location.pathname).toEqual('/mapview'));
    });
  });
});
