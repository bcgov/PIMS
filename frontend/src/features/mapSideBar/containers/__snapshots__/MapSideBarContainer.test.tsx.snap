// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parcel Detail MapSideBarContainer basic data loading and display Empty parcel sidebar matches snapshot 1`] = `
<div
  class="Toastify"
/>
`;

exports[`Parcel Detail MapSideBarContainer basic data loading and display parcel sidebar snapshot loads by id 1`] = `
"<div class=\\"Toastify\\"></div>
<div class=\\"map-side-drawer show\\">
  <div>
    <div class=\\"sc-fzqBkg eJxuZW\\"><span class=\\"sc-fzoxKX ddavXc mr-auto\\">Submit a Property</span><svg stroke=\\"currentColor\\" fill=\\"currentColor\\" stroke-width=\\"0\\" viewBox=\\"0 0 512 512\\" class=\\"sc-fzqPZZ eEQrNa\\" height=\\"1em\\" width=\\"1em\\" xmlns=\\"http://www.w3.org/2000/svg\\">
        <title>close</title>
        <path d=\\"M464 32H48C21.5 32 0 53.5 0 80v352c0 26.5 21.5 48 48 48h416c26.5 0 48-21.5 48-48V80c0-26.5-21.5-48-48-48zm-83.6 290.5c4.8 4.8 4.8 12.6 0 17.4l-40.5 40.5c-4.8 4.8-12.6 4.8-17.4 0L256 313.3l-66.5 67.1c-4.8 4.8-12.6 4.8-17.4 0l-40.5-40.5c-4.8-4.8-4.8-12.6 0-17.4l67.1-66.5-67.1-66.5c-4.8-4.8-4.8-12.6 0-17.4l40.5-40.5c4.8-4.8 12.6-4.8 17.4 0l66.5 67.1 66.5-67.1c4.8-4.8 12.6-4.8 17.4 0l40.5 40.5c4.8 4.8 4.8 12.6 0 17.4L313.3 256l67.1 66.5z\\"></path>
      </svg></div>
    <div class=\\"parcelDetailForm row no-gutters\\">
      <div class=\\"col\\">
        <form novalidate=\\"\\" class=\\"\\">
          <div class=\\"sc-fznzOf fQMnRX form-controls\\"><button type=\\"submit\\" class=\\"btn btn-primary\\">Submit</button></div><button data-testid=\\"delete\\" type=\\"button\\" class=\\"delete-btn btn btn-primary\\"><svg stroke=\\"currentColor\\" fill=\\"currentColor\\" stroke-width=\\"0\\" viewBox=\\"0 0 448 512\\" size=\\"20\\" height=\\"20\\" width=\\"20\\" xmlns=\\"http://www.w3.org/2000/svg\\">
              <path d=\\"M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z\\"></path>
            </svg></button>
          <div class=\\"container\\"></div>
          <div class=\\"container\\"></div>
          <nav class=\\"nav nav-tabs\\" role=\\"tablist\\"><a href=\\"#\\" role=\\"tab\\" data-rb-event-key=\\"parcel\\" id=\\"approvalTabs-tab-parcel\\" aria-controls=\\"approvalTabs-tabpane-parcel\\" aria-selected=\\"true\\" class=\\"nav-item nav-link active\\">Parcel</a><a href=\\"#\\" role=\\"tab\\" data-rb-event-key=\\"buildings\\" id=\\"approvalTabs-tab-buildings\\" aria-controls=\\"approvalTabs-tabpane-buildings\\" tabindex=\\"-1\\" aria-selected=\\"false\\" class=\\"nav-item nav-link\\">Buildings</a></nav>
          <div class=\\"tab-content\\">
            <div id=\\"approvalTabs-tabpane-parcel\\" aria-labelledby=\\"approvalTabs-tab-parcel\\" role=\\"tabpanel\\" aria-hidden=\\"false\\" class=\\"fade tab-pane active show\\">
              <div class=\\"row no-gutters\\">
                <div class=\\"col-md-6\\">
                  <div class=\\"d-inline-flex flex-nowrap form-row\\"><label class=\\"req form-label\\">*</label>PID<div class=\\"form-group\\"><input name=\\"pid\\" id=\\"input-pid\\" class=\\"input-small form-control\\" value=\\"000-000-000\\"></div><label class=\\"form-label\\" style=\\"width: 35px; min-width: 35px; padding-left: 5px;\\">or<br>PIN&nbsp;</label>
                    <div class=\\"form-group\\"><input name=\\"pin\\" type=\\"number\\" id=\\"input-pin\\" class=\\"input-small form-control\\" value=\\"\\"><svg stroke=\\"currentColor\\" fill=\\"currentColor\\" stroke-width=\\"0\\" viewBox=\\"0 0 512 512\\" class=\\"tooltip-icon\\" height=\\"1em\\" width=\\"1em\\" xmlns=\\"http://www.w3.org/2000/svg\\">
                        <path d=\\"M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z\\"></path>
                      </svg></div>
                  </div>
                  <div class=\\"form-row\\"><label class=\\"form-label\\">Name</label>
                    <div class=\\"form-group\\"><input name=\\"name\\" id=\\"input-name\\" class=\\"form-control\\" value=\\"test name\\"></div>
                  </div>
                  <div class=\\"form-row\\"><label class=\\"form-label\\">Description</label>
                    <div class=\\"form-group\\"><textarea name=\\"description\\" id=\\"input-description\\" class=\\"form-control\\">test</textarea></div>
                  </div>
                  <div class=\\"form-row\\"><label class=\\"form-label\\">Classification</label>
                    <div class=\\"form-group\\"><select name=\\"classificationId\\" type=\\"number\\" id=\\"input-classificationId-fast\\" class=\\"form-control\\">
                        <option value=\\"\\">Must Select One</option>
                      </select></div>
                  </div>
                  <div class=\\"form-row\\"><label class=\\"form-label\\">Agency</label>
                    <div class=\\"AutoCompleteText\\">
                      <div class=\\"form-group\\">
                        <div class=\\"input-area\\"><input name=\\"agencyId\\" id=\\"input-agencyId\\" class=\\"form-control\\" value=\\"\\"></div>
                      </div>
                    </div>
                  </div>
                </div>
                <div class=\\"col-md-6\\">
                  <div class=\\"field-row row\\">
                    <p class=\\"label\\"><span class=\\"req\\">*</span>Street Address</p>
                    <div class=\\"GeocoderAutoComplete\\">
                      <div>
                        <div class=\\"form-group\\"><input name=\\"address.line1\\" field=\\"address.line1\\" id=\\"input-address.line1\\" class=\\"form-control\\" value=\\"1234 mock Street\\"></div>
                      </div>
                    </div>
                  </div>
                  <div class=\\"field-row row\\">
                    <p class=\\"label\\"><span class=\\"req\\">*</span>Location</p>
                    <div class=\\"sc-AxjAm gLmrxD form-group\\">
                      <div class=\\"rbt\\" style=\\"outline: none; position: relative;\\" tabindex=\\"-1\\">
                        <div style=\\"display: flex; flex: 1; height: 100%; position: relative;\\"><input autocomplete=\\"off\\" type=\\"text\\" name=\\"address.administrativeArea\\" id=\\"address.administrativeArea-field\\" aria-autocomplete=\\"both\\" aria-expanded=\\"false\\" aria-haspopup=\\"listbox\\" role=\\"combobox\\" class=\\"rbt-input-main form-control rbt-input\\" value=\\"Victoria\\"><input aria-hidden=\\"true\\" class=\\"rbt-input-hint\\" readonly=\\"\\" style=\\"background-color: transparent; border-color: transparent; box-shadow: none; color: rgba(0, 0, 0, 0.35); left: 0px; pointer-events: none; position: absolute; top: 0px; width: 100%;\\" tabindex=\\"-1\\" value=\\"\\"></div>
                      </div>
                    </div>
                  </div>
                  <div class=\\"field-row row\\">
                    <p class=\\"label\\">Province</p>
                    <div class=\\"form-group\\"><select name=\\"address.provinceId\\" disabled=\\"\\" id=\\"input-address.provinceId\\" class=\\"form-select form-control\\">
                        <option value=\\"\\">Must Select One</option>
                      </select></div>
                  </div>
                  <div class=\\"postal row\\">
                    <p class=\\"label\\">Postal Code</p>
                    <div class=\\"form-group\\"><input name=\\"address.postal\\" style=\\"width: 120px;\\" id=\\"input-address.postal\\" class=\\"input-small form-control\\" value=\\"V1V1V1\\"></div>
                  </div>
                  <div class=\\"lat-long\\" style=\\"position: relative;\\">
                    <div class=\\"row\\">
                      <div class=\\"col-md-9\\">
                        <div class=\\"instruction\\">
                          <p>Drag and drop the pin on the map to mark the location of this parcel, or if you already have the coordinates, you can enter them manually in the fields below.</p>
                        </div>
                      </div>
                      <div class=\\"marker-svg col\\"><button class=\\"sc-fzoKki bhbZHm\\"><svg class=\\"parcel-icon\\">draft-parcel-icon.svg</svg></button></div>
                    </div>
                    <div class=\\"form-row\\">
                      <p class=\\"label\\"><span class=\\"req\\">*</span>Latitude</p>
                      <div class=\\"form-group\\"><input name=\\"latitude\\" type=\\"number\\" id=\\"input-latitude\\" class=\\"input-medium form-control\\" value=\\"48\\"></div>
                    </div>
                    <div class=\\"form-row\\">
                      <p class=\\"label\\"><span class=\\"req\\">*</span>Longitude</p>
                      <div class=\\"form-group\\"><input name=\\"longitude\\" type=\\"number\\" id=\\"input-longitude\\" class=\\"input-medium form-control\\" value=\\"123\\"></div>
                    </div>
                  </div>
                  <div class=\\"form-row\\"><label class=\\"required form-label\\">Lot Size</label>
                    <div class=\\"input-group\\">
                      <div class=\\"input-group-content\\">
                        <div class=\\"form-group\\"><input name=\\"landArea\\" type=\\"number\\" id=\\"input-landArea\\" class=\\"form-control\\" value=\\"123\\"></div>
                      </div>
                      <div class=\\"input-group-append\\"><span class=\\"input-group-text\\">Hectares</span></div>
                    </div>
                  </div>
                  <div class=\\"form-row\\"><label class=\\"form-label\\">Current Zoning</label>
                    <div class=\\"form-group\\"><input name=\\"zoning\\" id=\\"input-zoning\\" class=\\"form-control\\" value=\\"\\"></div>
                  </div>
                  <div class=\\"form-row\\"><label class=\\"form-label\\">Potential Zoning</label>
                    <div class=\\"form-group\\"><input name=\\"zoningPotential\\" id=\\"input-zoningPotential\\" class=\\"form-control\\" value=\\"\\"></div>
                  </div>
                </div>
              </div>
              <div class=\\"scroll\\">
                <div class=\\"row no-gutters\\">
                  <div class=\\"col\\">
                    <h4>Valuation Information</h4>
                    <p>Total values for parcel inclusive of existing building(s) for the most recent year with data</p>
                    <div class=\\"row no-gutters\\">
                      <div class=\\"col-md-6\\">
                        <div class=\\"form-row\\"><label class=\\"form-label\\">Assessed Sum</label>
                          <div class=\\"fast-currency-input form-group\\">
                            <div class=\\"input-tooltip-wrapper\\"><input name=\\"Assessed\\" class=\\"form-control\\" disabled=\\"\\" placeholder=\\"$0\\" value=\\"10000\\"></div>
                          </div>
                        </div>
                      </div>
                      <div class=\\"col-md-6\\">
                        <div class=\\"form-row\\"><label class=\\"form-label\\">Net Book Sum</label>
                          <div class=\\"fast-currency-input form-group\\">
                            <div class=\\"input-tooltip-wrapper\\"><input name=\\"NetBook\\" class=\\"form-control\\" disabled=\\"\\" placeholder=\\"$0\\" value=\\"10000\\"></div>
                          </div>
                        </div>
                        <div class=\\"form-row\\"><label class=\\"form-label\\">Market Sum</label>
                          <div class=\\"fast-currency-input form-group\\">
                            <div class=\\"input-tooltip-wrapper\\"><input name=\\"Market\\" class=\\"form-control\\" disabled=\\"\\" placeholder=\\"$0\\" value=\\"\\"></div>
                          </div>
                        </div>
                      </div>
                    </div>
                    <div>
                      <div>
                        <div role=\\"table\\" class=\\"table\\">
                          <div class=\\"thead thead-light\\">
                            <div role=\\"row\\" style=\\"display: flex; flex: 1 0 auto; min-width: 240px;\\" class=\\"tr\\">
                              <div colspan=\\"4\\" role=\\"columnheader\\" style=\\"box-sizing: border-box; flex: 350 0 auto; min-width: 120px; width: 350px; justify-content: flex-start; text-align: left; align-items: center; display: flex;\\" class=\\"th\\">Assessed</div>
                              <div colspan=\\"2\\" role=\\"columnheader\\" style=\\"box-sizing: border-box; flex: 150 0 auto; min-width: 60px; width: 150px; justify-content: flex-start; text-align: left; align-items: center; display: flex;\\" class=\\"th\\">Net Book Value</div>
                              <div colspan=\\"2\\" role=\\"columnheader\\" style=\\"box-sizing: border-box; flex: 150 0 auto; min-width: 60px; width: 150px; justify-content: flex-start; text-align: left; align-items: center; display: flex;\\" class=\\"th\\">Estimated Market Value</div>
                            </div>
                            <div role=\\"row\\" style=\\"display: flex; flex: 1 0 auto; min-width: 240px;\\" class=\\"tr\\">
                              <div colspan=\\"1\\" role=\\"columnheader\\" style=\\"box-sizing: border-box; flex: 50 0 auto; min-width: 30px; width: 50px; justify-content: flex-start; text-align: left; align-items: center; display: flex;\\" class=\\"th\\">Year</div>
                              <div colspan=\\"1\\" role=\\"columnheader\\" style=\\"box-sizing: border-box; flex: 100 0 auto; min-width: 30px; width: 100px; justify-content: flex-start; text-align: left; align-items: center; display: flex;\\" class=\\"th\\">Land</div>
                              <div colspan=\\"1\\" role=\\"columnheader\\" style=\\"box-sizing: border-box; flex: 100 0 auto; min-width: 30px; width: 100px; justify-content: flex-start; text-align: left; align-items: center; display: flex;\\" class=\\"th\\">Improvements</div>
                              <div colspan=\\"1\\" role=\\"columnheader\\" style=\\"box-sizing: border-box; flex: 100 0 auto; min-width: 30px; width: 100px; justify-content: flex-start; text-align: left; align-items: center; display: flex;\\" class=\\"th\\">Total</div>
                              <div colspan=\\"1\\" role=\\"columnheader\\" style=\\"box-sizing: border-box; flex: 50 0 auto; min-width: 30px; width: 50px; justify-content: flex-start; text-align: left; align-items: center; display: flex;\\" class=\\"th\\">Fiscal Year</div>
                              <div colspan=\\"1\\" role=\\"columnheader\\" style=\\"box-sizing: border-box; flex: 100 0 auto; min-width: 30px; width: 100px; justify-content: flex-start; text-align: left; align-items: center; display: flex;\\" class=\\"th\\">Value</div>
                              <div colspan=\\"1\\" role=\\"columnheader\\" style=\\"box-sizing: border-box; flex: 50 0 auto; min-width: 30px; width: 50px; justify-content: flex-start; text-align: left; align-items: center; display: flex;\\" class=\\"th\\">Fiscal Year</div>
                              <div colspan=\\"1\\" role=\\"columnheader\\" style=\\"box-sizing: border-box; flex: 100 0 auto; min-width: 30px; width: 100px; justify-content: flex-start; text-align: left; align-items: center; display: flex;\\" class=\\"th\\">Value</div>
                            </div>
                          </div>
                          <div role=\\"rowgroup\\" class=\\"tbody\\">
                            <div class=\\"tr-wrapper\\">
                              <div role=\\"row\\" style=\\"display: flex; flex: 1 0 auto; min-width: 240px;\\" class=\\"tr\\">
                                <div role=\\"cell\\" style=\\"box-sizing: border-box; flex: 50 0 auto; min-width: 30px; width: 50px; justify-content: flex-start; text-align: left; align-items: flex-start; display: flex;\\" title=\\"\\" class=\\"td\\">2021</div>
                                <div role=\\"cell\\" style=\\"box-sizing: border-box; flex: 100 0 auto; min-width: 30px; width: 100px; justify-content: flex-start; text-align: left; align-items: flex-start; display: flex;\\" title=\\"\\" class=\\"td\\">N/A</div>
                                <div role=\\"cell\\" style=\\"box-sizing: border-box; flex: 100 0 auto; min-width: 30px; width: 100px; justify-content: flex-start; text-align: left; align-items: flex-start; display: flex;\\" title=\\"\\" class=\\"td\\"></div>
                                <div role=\\"cell\\" style=\\"box-sizing: border-box; flex: 100 0 auto; min-width: 30px; width: 100px; justify-content: flex-start; text-align: left; align-items: flex-start; display: flex;\\" title=\\"\\" class=\\"td\\"></div>
                                <div role=\\"cell\\" style=\\"box-sizing: border-box; flex: 50 0 auto; min-width: 30px; width: 50px; justify-content: flex-start; text-align: left; align-items: flex-start; display: flex;\\" title=\\"\\" class=\\"td\\">20/21</div>
                                <div role=\\"cell\\" style=\\"box-sizing: border-box; flex: 100 0 auto; min-width: 30px; width: 100px; justify-content: flex-start; text-align: left; align-items: flex-start; display: flex;\\" title=\\"\\" class=\\"td\\">
                                  <div class=\\"fast-currency-input form-group\\">
                                    <div class=\\"input-tooltip-wrapper\\"><input name=\\"financials.0.netbook.value\\" class=\\"form-control\\" placeholder=\\"\\" value=\\"\\"></div>
                                  </div>
                                </div>
                                <div role=\\"cell\\" style=\\"box-sizing: border-box; flex: 50 0 auto; min-width: 30px; width: 50px; justify-content: flex-start; text-align: left; align-items: flex-start; display: flex;\\" title=\\"\\" class=\\"td\\">20/21</div>
                                <div role=\\"cell\\" style=\\"box-sizing: border-box; flex: 100 0 auto; min-width: 30px; width: 100px; justify-content: flex-start; text-align: left; align-items: flex-start; display: flex;\\" title=\\"\\" class=\\"td\\">
                                  <div class=\\"fast-currency-input form-group\\">
                                    <div class=\\"input-tooltip-wrapper\\"><input name=\\"financials.0.market.value\\" class=\\"form-control\\" placeholder=\\"\\" value=\\"\\"></div>
                                  </div>
                                </div>
                              </div>
                            </div>
                            <div class=\\"tr-wrapper\\">
                              <div role=\\"row\\" style=\\"display: flex; flex: 1 0 auto; min-width: 240px;\\" class=\\"tr\\">
                                <div role=\\"cell\\" style=\\"box-sizing: border-box; flex: 50 0 auto; min-width: 30px; width: 50px; justify-content: flex-start; text-align: left; align-items: flex-start; display: flex;\\" title=\\"\\" class=\\"td\\">2020</div>
                                <div role=\\"cell\\" style=\\"box-sizing: border-box; flex: 100 0 auto; min-width: 30px; width: 100px; justify-content: flex-start; text-align: left; align-items: flex-start; display: flex;\\" title=\\"\\" class=\\"td\\">
                                  <div class=\\"fast-currency-input form-group\\">
                                    <div class=\\"input-tooltip-wrapper\\"><input name=\\"financials.1.assessed.value\\" class=\\"form-control\\" placeholder=\\"\\" value=\\"10000\\"></div>
                                  </div>
                                </div>
                                <div role=\\"cell\\" style=\\"box-sizing: border-box; flex: 100 0 auto; min-width: 30px; width: 100px; justify-content: flex-start; text-align: left; align-items: flex-start; display: flex;\\" title=\\"\\" class=\\"td\\"></div>
                                <div role=\\"cell\\" style=\\"box-sizing: border-box; flex: 100 0 auto; min-width: 30px; width: 100px; justify-content: flex-start; text-align: left; align-items: flex-start; display: flex;\\" title=\\"\\" class=\\"td\\">$10,000</div>
                                <div role=\\"cell\\" style=\\"box-sizing: border-box; flex: 50 0 auto; min-width: 30px; width: 50px; justify-content: flex-start; text-align: left; align-items: flex-start; display: flex;\\" title=\\"\\" class=\\"td\\">19/20</div>
                                <div role=\\"cell\\" style=\\"box-sizing: border-box; flex: 100 0 auto; min-width: 30px; width: 100px; justify-content: flex-start; text-align: left; align-items: flex-start; display: flex;\\" title=\\"\\" class=\\"td\\">
                                  <div class=\\"fast-currency-input form-group\\">
                                    <div class=\\"input-tooltip-wrapper\\"><input name=\\"financials.1.netbook.value\\" class=\\"form-control\\" placeholder=\\"\\" value=\\"10000\\"></div>
                                  </div>
                                </div>
                                <div role=\\"cell\\" style=\\"box-sizing: border-box; flex: 50 0 auto; min-width: 30px; width: 50px; justify-content: flex-start; text-align: left; align-items: flex-start; display: flex;\\" title=\\"\\" class=\\"td\\">19/20</div>
                                <div role=\\"cell\\" style=\\"box-sizing: border-box; flex: 100 0 auto; min-width: 30px; width: 100px; justify-content: flex-start; text-align: left; align-items: flex-start; display: flex;\\" title=\\"\\" class=\\"td\\">
                                  <div class=\\"fast-currency-input form-group\\">
                                    <div class=\\"input-tooltip-wrapper\\"><input name=\\"financials.1.market.value\\" class=\\"form-control\\" placeholder=\\"\\" value=\\"\\"></div>
                                  </div>
                                </div>
                              </div>
                            </div>
                          </div>
                        </div>
                        <div class=\\"table-toolbar\\">
                          <ul class=\\"pagination\\">
                            <li class=\\"page-item disabled\\"><a class=\\"page-link\\" tabindex=\\"0\\" role=\\"button\\" aria-disabled=\\"true\\">&lt;</a></li>
                            <li class=\\"page-item active\\"><a role=\\"button\\" class=\\"page-link\\" tabindex=\\"0\\" aria-label=\\"Page 1 is your current page\\" aria-current=\\"page\\">1</a></li>
                            <li class=\\"page-item\\"><a role=\\"button\\" class=\\"page-link\\" tabindex=\\"0\\" aria-label=\\"Page 2\\">2</a></li>
                            <li class=\\"page-item\\"><a role=\\"button\\" class=\\"page-link\\" tabindex=\\"0\\" aria-label=\\"Page 3\\">3</a></li>
                            <li class=\\"page-item\\"><a role=\\"button\\" class=\\"page-link\\" tabindex=\\"0\\" aria-label=\\"Page 4\\">4</a></li>
                            <li class=\\"page-item\\"><a role=\\"button\\" class=\\"page-link\\" tabindex=\\"0\\" aria-label=\\"Page 5\\">5</a></li>
                            <li class=\\"page-item\\"><a role=\\"button\\" class=\\"page-link\\" tabindex=\\"0\\" aria-label=\\"Page 6\\">6</a></li>
                            <li class=\\"page-item\\"><a class=\\"page-link\\" role=\\"button\\" tabindex=\\"0\\">...</a></li>
                            <li class=\\"page-item\\"><a role=\\"button\\" class=\\"page-link\\" tabindex=\\"0\\" aria-label=\\"Page 9\\">9</a></li>
                            <li class=\\"page-item\\"><a role=\\"button\\" class=\\"page-link\\" tabindex=\\"0\\" aria-label=\\"Page 10\\">10</a></li>
                            <li class=\\"page-item\\"><a class=\\"page-link\\" tabindex=\\"0\\" role=\\"button\\" aria-disabled=\\"false\\">&gt;</a></li>
                          </ul>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </form>
      </div>
    </div>
  </div>
</div>"
`;

exports[`Parcel Detail MapSideBarContainer basic data loading and display sidebar is hidden 1`] = `
<div
  class="Toastify"
/>
`;
